src/C4Config.cpp

```diff
@@ -387,14 +389,22 @@ bool C4Config::Load(bool forceWorkingDirectory, const char *szConfigFile)
 			}
 			else
 			{
+#ifdef __APPLE__
```

Comment:
This isn't apple-specific stuff, just code which doesn't use XDG. BSD would use the wrong code path as well.

Reply:
I initially had different code here, looks like I messed up restoring it to its previous form here.